### PR TITLE
Process plotting

### PR DESCRIPTION
**Pull request by schaefes**
Added a function to filter on signal specific jets in test set. Done through a comparison of the test set and the signal set. Signal jets must be processed in the same fashion as the test set (make_data, load_data and to_ML). The signal data is created along the with the training data using --make-data and -sig (list of signal files) to specify which signal are of interest for plotting. The filtering is done in filter_processes through np.unique to identify the correct jets through the counts.

Signal processes must be added in CI/ml_pipeline.yml for the signal variable and in process_labels() in tagger/plot/basic.py

### CODE REVIEW

**Comment by Chriisbrown** [comment]
Could you remove the signal_process_data folder from the git index to avoid the process data root files being added to the repository, if the inp_masking and masking folders are created during the training could you also remove these too. All that should be in the repo are the files needed to run the code and none of the files created when the code is run
